### PR TITLE
Update omsadmin.sh for LAD onboarding

### PR DESCRIPTION
**Pull request by hosungsmsft**
Per our discussion at our 3/15/2017 meeting. This is all we need for now.

Thanks Laura for the SCOM pointer. It was very helpful.

### CODE REVIEW

**Comment by lagalbra** [comment]
@Microsoft/omsagent-devs

File: installer/scripts/omsadmin.sh [hunk 1]
```diff
@@ -349,6 +349,21 @@ onboard_scom()
     make_dir $CONF_DIR/omsagent.d
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
+    save_config
```
Comment (lagalbra) [inline installer/scripts/omsadmin.sh]: Why did you add a save_config call to both onboard_scom and onboard_lad? That method writes to `omsadmin.conf`, which is not an appropriately named file for you to store your variables.

Comment (hosungsmsft) [inline installer/scripts/omsadmin.sh]: Thanks. I'll remove this call from onboard_scom (I shouldn't have assumed anyway), but for LAD, I wasn't able to run the service_control script without creating omsadmin.conf, so that's why I added. I still would like to rely on the service_control script to start/stop omsagent, so I thought I need omsadmin.conf anyway. Please correct me if I'm wrong.

File: installer/scripts/omsadmin.sh [hunk 2]
```diff
@@ -349,6 +349,21 @@ onboard_scom()
...
+    configure_logrotate
+}
+
+onboard_lad()
+{
+    echo "Onboarding LAD"
+    create_workspace_directories "LAD"
+
+    echo "@include omsagent.d/*" > $CONF_DIR/omsagent.conf
```
Comment (hosungsmsft) [inline installer/scripts/omsadmin.sh]: Will do. Thanks.

**Comment by lagalbra** [comment]
If you rely on service_control to start/stop an omsagent daemon (which I think you may have to create a copy of, given that Kris said the daemon points to the FluentD root .conf file which you must populate), you have to add this line, like `onboard()` has:
`$SERVICE_CONTROL start "LAD"`
Will you be including this piece in a separate PR?

File: installer/scripts/omsadmin.sh (hunk 1 above)
Comment (lagalbra) [inline installer/scripts/omsadmin.sh]: Yes, currently service_control relies on omsadmin.conf being present.

**Comment by hosungsmsft** [comment]
No, I don't think I'll be including the line ('service_control start LAD') in omsadmin.sh in this or a separate PR. Currently omsagent.conf and omsagent.d/* are blank (except '@include omsagent.d/*' in omsagent.conf) after initial onboarding, so I don't think it makes sense to start the service from omsadmin.sh. I'm thinking of calling 'service_control start LAD' separately from LAD extension, after configuring omsagent.d/* according to the passed LAD settings.
